### ISSUE TITLE
Training Data

### ISSUE THREAD

**Issue by pcjedi** [haotianteng/Chiron#63]
I found the supporting data at gigadb.org (http://gigadb.org/dataset/100425, which is falsely linked at academic.oup.com, by the way).
I find signal and label data, but a mandatory tfrecords file appears to be missing, or can I use any tfrecord file?

**Comment by haotianteng** [comment]
The tfrecords file is generated if you follow the steps described in
README.md.

**Comment by pcjedi** [comment]
Thanks, but I wanted to train Chiron with the 'original' trainingdata provided by gigadb. The signal and label data provided via gigadb appears to be useless unless the corresponding tfrecords file is also provided.

**Comment by haotianteng** [comment]
Oh yes that's correct as Chiron no longer use the signal and label file
format, you can do it in two ways, one is to use the older version of
Chiron, e.g. 0.3, or you can download the original fast5 files from
https://data.genomicsresearch.org/Projects/train_set_all

**Comment by SrinikhilReddy** [comment]
Hi @haotianteng. Wanted to know if the model provided in chiron/model/DNA_Default was entirely trained using the dataset in http://gigadb.org/dataset/100425  or if any additional training sets were used. If yes, could me point me to the same resources? 

Thanks,
Naga.

**Comment by haotianteng** [comment]
Chiron V0.3 is entirely trained using the dataset described in the paper. No additional training sets were used. Chiron V0.4, on the other hands, use an additional human dataset to make it perform better on Human dataset.
